python/cugraph/cugraph/dask/link_analysis/hits.py

```diff
@@ -0,0 +1,186 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_distributed_data
+
+import cugraph.comms.comms as Comms
+import dask_cudf
+
+import pylibcugraph
+
+
+def call_hits(sID,
+              data,
+              src_col_name,
+              dst_col_name,
+              graph_properties,
+              store_transposed,
+              num_edges,
+              do_expensive_check,
+              tolerance,
+              max_iter,
+              initial_hubs_guess_vertices,
+              initial_hubs_guess_value,
+              normalized):
+
+    handle = Comms.get_handle(sID)
+    h = pylibcugraph.experimental.ResourceHandle(handle.getHandle())
+    srcs = data[0][src_col_name]
+    dsts = data[0][dst_col_name]
+    weights = None
+    if "value" in data[0].columns:
+        weights = data[0]['value']
+
+    mg = pylibcugraph.experimental.MGGraph(h,
+                                           graph_properties,
+                                           srcs,
+                                           dsts,
+                                           weights,
+                                           store_transposed,
+                                           num_edges,
+                                           do_expensive_check)
+
+    result = pylibcugraph.experimental.hits(h,
+                                            mg,
+                                            tolerance,
+                                            max_iter,
+                                            initial_hubs_guess_vertices,
+                                            initial_hubs_guess_value,
+                                            normalized,
+                                            do_expensive_check)
+
+    return result
+
+
+def hits(input_graph, tol=1.0e-5, max_iter=100,  nstart=None, normalized=True):
+    """
+    Compute HITS hubs and authorities values for each vertex
+
+    The HITS algorithm computes two numbers for a node.  Authorities
+    estimates the node value based on the incoming links.  Hubs estimates
+    the node value based on outgoing links.
+
+    The cuGraph implementation of HITS is a wrapper around the gunrock
+    implementation of HITS.
+
+    Note that the gunrock implementation uses a 2-norm, while networkx
+    uses a 1-norm.  The raw scores will be different, but the rank ordering
+    should be comparable with networkx.
+
```

Comment:
The reference to "Gunrock" should be dropped since we are no longer using that as the backend.   Does our version use 1- or 2-norm?

Reply:
I dropped the "Gunrock" reference.  regarding the implementation , I will ask Chuck and update the docstring accordingly

Reply:
The Gunrock implementation used the L2 norm.  Our primitive based implementation uses the L1 norm.

Reply:
Fixed